src/Style/Sass/base/_index.scss

```diff
@@ -0,0 +1,5 @@
+button,
+input {
+  outline: none;
+  background-color: #ebebeb;
```

Comment:
don't you think that this one will be added to any button and input?? use a personalized class instead

Reply:
I have removed the file it is not needed anyway